packages/gestalt/src/Typeahead.js

```diff
@@ -0,0 +1,301 @@
+// @flow strict
+import * as React from 'react';
+import { useState, useRef, type Node } from 'react';
+import PropTypes from 'prop-types';
+import TypeaheadInputField from './TypeaheadInputField.js';
+import Option from './TypeaheadOption.js';
+import Box from './Box.js';
+import Text from './Text.js';
+import Flyout from './Flyout.js';
+import Layer from './Layer.js';
+
+type OptionObject = {|
+  label: string,
+  value: string,
+|};
+
+type Props = {|
+  data: Array<{
+    label: string,
+    value: string,
+  }>,
+  defaultItem?: OptionObject,
```

Comment:
@simkessy here `defaultItem` is optional but in the docs it is required. Could you make sure the docs and the code align?